com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Configuration.cs

```diff
@@ -693,9 +699,12 @@ void CheckPersistantNeedReboot()
         }
 
         bool IsDXRResourcesCorrect()
-            => IsHdrpGlobalSettingsUsedCorrect()
-            && HDRenderPipelineGlobalSettings.instance.AreRayTracingResourcesCreated()
-            && SystemInfo.supportsRayTracing;
+        {
+            var selectedBuildTarget = CalculateSelectedBuildTarget();
+            return IsHdrpGlobalSettingsUsedCorrect()
+                && HDRenderPipelineGlobalSettings.instance.AreRayTracingResourcesCreated()
+                && (SystemInfo.supportsRayTracing || selectedBuildTarget == BuildTarget.PS5);
```

Comment:
can't we have SystemInfo.supportsRayTracing returning true on PS5 instead of having this separate test (selectedBuildTarget == BuildTarget.PS5) ?

Reply:
The issue was when building a player for PS5, on a non RTX PC the check would fail.